loadtest/user/userentity/user.go

```diff
@@ -96,6 +97,21 @@ func (ue *UserEntity) Connect() <-chan error {
 	return ue.wsErrorChan
 }
 
+// FetchStaticAssets parses index.html and fetches static assets mentioned in link/script tags.
+func (ue *UserEntity) FetchStaticAssets() error {
+	c := colly.NewCollector(colly.MaxDepth(1))
+
+	c.OnHTML("link[href]", func(e *colly.HTMLElement) {
+		link := e.Attr("href")
+		c.Visit(e.Request.AbsoluteURL(link))
+	})
+	c.OnHTML("script[src]", func(e *colly.HTMLElement) {
+		link := e.Attr("src")
+		c.Visit(e.Request.AbsoluteURL(link))
+	})
+	return c.Visit(ue.client.Url)
+}
```

Comment:
My concern here is that on a real browser most of the stuff is downloaded after the main JS is parsed and executed.
Maybe we can hard-code a few things that we cannot figure out by just looking at the static html.
One example would be fetching config and license.

These are some calls that we make:
```
http://localhost:8065/api/v4/config/client?format=old
http://localhost:8065/api/v4/license/client?format=old
http://localhost:8065/api/v4/plugins/webapp
```

And then as consequence we also fetch all the plugins bundles.

I am okay if you prefer addressing this in a new PR but we need to think about it otherwise we are are not doing a great job at simulating this behaviour.

Reply:
Why can't we just do a maxDepth of 2 if we want to visit links inside the JS ?

Reply:
> Why can't we just do a maxDepth of 2 if we want to visit links inside the JS ?

Mmm. Does it parse JS? I am thinking there might be concatenated URLs or other non primitive format. We can surely try though and see how much it scrapes.

Reply:
Guys, I think you are digging too deep :) I sincerely doubt that the static asset fetching is our bottleneck...